pkgs/development/tools/build-managers/conan/default.nix

```diff
@@ -57,24 +49,26 @@ in newPython.pkgs.buildPythonApplication rec {
     mock
     webtest
     codecov
+    tox
   ]);
 
   propagatedBuildInputs = with newPython.pkgs; [
     requests fasteners pyyaml pyjwt colorama patch
     bottle pluginbase six distro pylint node-semver
-    future pygments mccabe deprecation
+    future pygments mccabe deprecation tqdm
   ];
 
   checkPhase = ''
     export HOME="$TMP/conan-home"
     mkdir -p "$HOME"
-    nosetests conans.test
+    tox conans.test
```

Comment:
Don't use tox. Tox is a tool for testing with different Python versions, which is of no use to us. You can look up which tool tox will run in `tox.ini`.

Reply:
Oh ok, in that case the tests fail. This is mostly because they have requirements not related to this package, like `cmake`, `pip`, `go` and friends. I am removing the test phase entirely. It works on any reasonable system but not in isolation. In any case, the archlinux AUR builds also seem to skip the test phase without any repercussions.